src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -351,6 +351,8 @@ public struct StartConfiguration
             public bool StartInTestMode { get; set; }
             public IUpdateManager UpdateManager { get; set; }
             public ISchedulerThread SchedulerThread { get; set; }
+            public IAuthProvider AuthProvider { get; set; }
+            public string PackageManagerAddress { get; set; }
```

Comment:
This is passed in the `StartConfiguration` object and then set to a `readonly` property in `PackageManagerClient`.